vector-stores/operations.tsp

```diff
@@ -0,0 +1,246 @@
+import "@typespec/http";
```

Comment:
Given this is a brand new folder, can you please add a matching line to the `main.tsp` so we pick up the change?

I think that should then let us generate a "VectorStoresClient" -- which I'm not sure is how we want to surface it, but it'll get us the operation representation in code.

The other folders all already exist, so I think this is the only one left out.